src/main/java/com/palehorsestudios/alone/player/Player.java

```diff
@@ -1,13 +1,9 @@
 package com.palehorsestudios.alone.player;
 
 import com.google.common.collect.ImmutableSet;
-import com.palehorsestudios.alone.Food;
-import com.palehorsestudios.alone.Item;
-import com.palehorsestudios.alone.Result;
-import com.palehorsestudios.alone.Shelter;
+import com.palehorsestudios.alone.*;
```

Comment:
We should avoid using wildcard imports. Applies to `import java.util.*` below too.